drivers/timers/arch_alarm.c

```diff
@@ -45,6 +45,10 @@
 
 static FAR struct oneshot_lowerhalf_s *g_oneshot_lower;
 
+#ifndef CONFIG_SCHED_TICKLESS
+static clock_t g_current_tick;
```

Comment:
Do we need
```suggestion
static clock_t g_current_tick = 1;
```
?

Reply:
don't need since g_current_tick is initialized in up_alarm_set_lowerhalf